doc/en/user/source/extensions/netcdf/netcdf.rst

```diff
@@ -39,7 +39,7 @@ Configuring a NetCDF data store
 Notes on supported NetCDFs
 --------------------------
 
-The NetCDF plugin for GeoServer supports gridded NetCDF files having dimensions following the COARDS convention (custom, Time, Elevation, Lat, Lon).
+The NetCDF plugin for GeoServer supports gridded NetCDF files having dimensions following the COARDS convention (custom, Time, Elevation, Lat, Lon). The NetCDF plugin supports plain NetCDF datasets (.NC files) as well .NCML files (which aggregate and/or modify one or more datasets) and Feature Collections. It supports Forecast Model Run Collection Aggregations (FMRC) either through the NCML or Feature Collection syntax. It supports an unlimited amount of custom dimensions, including runtime. 
```

Comment:
Perhaps these extensions should be written ".nc" and ".ncml" as case is significant?